carbon-design-system/carbon
# feat(spacing): change layout variables to spacing

Closes https://github.com/carbon-design-system/carbon/issues/5602

Deprecates the `layout` tokens in favor of a single `spacing` scale. This change will need to coincide with a website change to update the new/removed tokens

https://www.carbondesignsystem.com/guidelines/spacing/overview#spacing-scale

https://www.carbondesignsystem.com/guidelines/spacing/overview#layout-scale

#### Changelog

**New**

- `$spacing-13` to match the largest `layout` variable

**Changed**

- Changed all internal references of `$layout` to `$spacing` (seemed to only be in Data Table styles)
- Added deprecation notes to old `layout` variables
- Added comment to remove `layout` from `theme` package in v11

#### Testing / Reviewing

- Ensure DataTable renders as expected
- Try using `$spacing-13`
